### ISSUE TITLE
What is the detection method used in the object proposal network to  proposal the rois?

### ISSUE THREAD

**Issue by skyforwork** [zhaoweicai/mscnn#72]
![default](https://cloud.githubusercontent.com/assets/25922773/26229225/e49366a2-3c72-11e7-8fd6-7b835ca2e942.png)
In the picture , What is the detection method used?

**Comment by zhaoweicai** [comment]
The detectors are just typical convolutional filters.

**Comment by skyforwork** [comment]
The output of the detectors is the bounding box and label?

**Comment by zhaoweicai** [comment]
Yes, "c" is the number of classes, and "b" the number of bounding box coordinates.

**Comment by skyforwork** [comment]
I am a novice. Can you teach me how does the detectors works in detail?Thank you very much.

**Comment by tomzhang666** [comment]
you can review fast-rcnn and faster-rcnn.
